link.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import argparse
+import os
+
+
+def main():
+    parser = argparse.ArgumentParser(description='link to source code')
+    parser.add_argument(
+        '--root',
+        help='root directory where you cloned the source code')
+    args = parser.parse_args()
+    if not args.root:
+        args.root = raw_input('What directory is your source code in? ')
+    args.root = os.path.normpath(os.path.expanduser(args.root))
+
+    errors = False
+    for name in [
+        'payments-service',
+        'solitude',
+    ]:
+        full_name = os.path.join(args.root, name)
+        if not os.path.exists(full_name):
+            print
+            print ('** Repository at {path} does not exist. '
+                   'Is it checked out?'.format(path=full_name))
+            print
+            errors = True
+            continue
+
+        dest = './docker/source-links/{name}'.format(name=name)
+        print '{source} -> {dest}'.format(source=full_name, dest=dest)
+
+        if os.path.exists(dest):
+            print '(already exists)'
+        else:
+            os.symlink(full_name, dest)
+
+    if errors:
+        parser.error('Not all symlinks were successful')
```

Comment:
I'm not crazy about the ``marketplace-env`` approach where you have a fig file in your home dir and you replace the paths in it. It means 1) you can't easily have multiple projects on your system, 2) the fig file is hard to update, and 3) there is an unnecessary level of indirection when you read the config file and imagine what the actual values will be. But, yeah, we could probably extract something out of the ``mkt`` command.